dotCMS/src/main/java/com/dotmarketing/servlets/ShortyServlet.java

```diff
@@ -357,24 +361,43 @@ private void addImagePath(final int width,
                             final Optional<FocalPoint> focalPoint,
                             final int cropWidth,
                             final int cropHeight,
-                            final int resampleOpt) {
+                            final int resampleOpt,
+                            final String[] filters) {
         if (isImage) {
-
-            if (quality > 0) {
-                pathBuilder.append("/quality_q/" + quality);
-            } else {
-                pathBuilder.append(jpeg ? "/jpeg_q/75" : StringPool.BLANK);
-                pathBuilder.append(webp ? "/webp_q/75" : StringPool.BLANK);
-                pathBuilder.append(jpeg && jpegp ? "/jpeg_p/1" : StringPool.BLANK);
+            for(String filter : filters){
```

Comment:
set to final

Reply:
it can't be final because it's being reuse on the next line:
`filter = StringPool.FORWARD_SLASH + filter;`